### DISCUSSION TITLE
rtp_mpegts output possible from gui?

### DISCUSSION THREAD

**Discussion by mihaipopoiag** [datarhei/restreamer#477] [Ideas]
Hello, can you please add in protocols rtp_mpegts option for rtp:// output? Thank you!

**Comment by jstabenow** [comment]
Are you referring to [prompeg](http://ffmpeg.org/ffmpeg-all.html#prompeg)?
Would be possible: `-f rtp_mpegts -fec prompeg=option=val... rtp://hostname:port`

Can you show some real world examples?

**Reply by mihaipopoiag** [reply]
Kind of, i have an old russian :) software encoder that accepts live inputs in this format and it is the only stable input i can pass to it this way.
Example:

ffmpeg -re -i rtmp://input_ip_address:1935/test/live?fifo_size=1000000&buffer_size=1000000 -vcodec copy -acodec copy -bufsize 512k -f rtp_mpegts rtp://192.168.1.204:2346?localaddr=out_ethernet_ip_address

With this the final encoder is starting and working, with another protocol it is not even starting, no matter if it is supported... I wanted to give it a try so i created an rtp stream from restreamer and then modified the code inside db.json with -f rtp_mpegts and it worked.

"options": [
                            "-map",
                            "0:0",
                            "-codec:v",
                            "copy",
                            "-map",
                            "0:1",
                            "-codec:a",
                            "copy",
                            "-f",
                            "rtp_mpegts",
